src/razor/src/semantic/semanticTokensRangeHandler.ts

```diff
@@ -16,9 +21,14 @@ export class SemanticTokensRangeHandler {
         ProvideSemanticTokensResponse,
         any
     > = new RequestType(SemanticTokensRangeHandler.getSemanticTokensRangeEndpoint);
-    private emptyTokensInResponse: Array<Array<number>> = new Array<Array<number>>();
+    private emptyTokensResponse: Uint32Array = new Uint32Array();
```

Comment:
❓ what is the benefit of `Uint32Array` over `Array<number>`?

Reply:
I changed the code to use `vscode-languageclient.SemanticTokens` in my latest push - hopefully it's clearer!